src/olympia/conf/stage/settings.py

```diff
@@ -149,3 +149,5 @@
     'amo-account-change-stage')
 
 VAMO_URL = 'https://versioncheck.allizom.org'
+
+CODE_MANAGER_URL = 'https://code.addons.allizom.org'
```

Comment:
I wish we didn't need to define this in each conf/.../settings.py file but that's some refactoring away 😞

Reply:
Why not? Each URL is unique for each environment.

Reply:
Ideally we'd just do `CODE_MANAGER_URL = 'https://code.' + DOMAIN` in settings_base.py but as we re-define `DOMAIN` in each `settings.py` we have to then (re-)define `CODE_MANAGER_URL` anyway.  It much harder to distinguish genuine differences between prod, stage & dev in the settings but it needs some serious and careful refactoring of all our settings files to not redefine `DOMAIN` and others settings each time.

Reply:
oh, I see what you mean. That makes sense. I updated the patch to use env vars, too.